code/modules/mob/living/carbon/human/species_types/supersoldier.dm

```diff
@@ -16,4 +16,24 @@
 		if(light_amount > 0) //if there's any light, heal
 			H.heal_overall_damage(1,1, 0, BODYPART_ORGANIC)
 			H.adjustToxLoss(-1)
-			H.adjustOxyLoss(-1)
\ No newline at end of file
+			H.adjustOxyLoss(-1)
+
+/datum/species/human/supersoldier/syndie
+	name = "Syndicate Super Soldier" //inherited from the real species, for health scanners and things
+	id = "supersoldier_syndie"
+	species_traits = list(NOTRANSSTING) //all of these + whatever we inherit from the real species
+	inherent_traits = list(TRAIT_NOLIMBDISABLE,TRAIT_NOHUNGER,TRAIT_NODISMEMBER,TRAIT_IGNORESLOWDOWN,TRAIT_IGNOREDAMAGESLOWDOWN,TRAIT_STUNIMMUNE,TRAIT_CONFUSEIMMUNE,TRAIT_SLEEPIMMUNE,TRAIT_VIRUSIMMUNE,TRAIT_THERMAL_VISION,TRAIT_STRONG_GRABBER,TRAIT_ALWAYS_CLEAN,TRAIT_FEARLESS)
+	mutanteyes = /obj/item/organ/eyes/night_vision
+	changesource_flags = MIRROR_BADMIN
+
+/datum/species/human/supersoldier/syndie/spec_life(mob/living/carbon/human/H)
+	if(H.stat == DEAD)
+		return
+	var/light_amount = 0 //how much light there is in the place, affects receiving nutrition and healing
+	if(isturf(H.loc)) //else, there's considered to be no light
+		var/turf/T = H.loc
+		light_amount = min(1,T.get_lumcount()) - 0.5
+		if(light_amount > 0) //if there's any light, heal
+			H.heal_overall_damage(0.25,0.25, 0, BODYPART_ORGANIC)
```

Comment:
Having a var/heal_modifier in /datum/species/human/supersoldier and then modifying /datum/species/human/supersoldier/syndie to have heal_modifier = 0.25 would be better than copying the code again.

Reply:
i think i managed to do that